_i18n/nl.yml

```diff
@@ -0,0 +1,314 @@
+global:
+  date: '%Y-%m-%d'
+  firo: Firo
+  yeth: "ja"
+  noo: "nee"
+  lelantus: Lelantus
+  title: Firo - Privacybehoudende cryptovaluta
+  
+titles:
+  home: Startpagina
+  download: Download
+  buyfiro: Koop Firo
+  tech: Research
+  team: Team
+  roadmap: Roadmap
+  faq: FAQ
+  community: Community
+  guides: Handleidingen
+  blog: Blog
+  branding: Branding
+  merchants: Merchants
+  
+navigation:
+  getfiro: Start met Firo
+  about: Over
+  community: Community
+  social: Sociaal
+  merchants: Merchants
+  guides: Handleidingen
+  blog: Blog
+  crowdfunding: Crowdfunding
+  download: Download portemonnees
+  buyfiro: Koop Firo
+  earnfiro: Verdien Firo
+  mine: Mijnen
+  tech: Research
+  team: Team
+  roadmap: Roadmap
+  faq: FAQ
+  fcs: Firo-crowdfundingsysteem
+  opencollective: Open Collective
+  
+footer:
+  resources: Bronnen
+  branding: Branding
+  vuln: Kwetsbaarheidsprogramma
+  explorer: Explorer
+  website: Website
+  privacy: Privacybeleid
+  tos: Servicevoorwaarden
+  newsletter: Nieuws
+  ifsub: Bekijk ons <a href="/blog/">Blog</a> en Telegram-kanaal en blijf op de hoogte van Firo-nieuws.
+  newsbutton: Lees nieuws
+  comm: Communicatie
+  feed: RSS-feed
+
+  
+home:
+  heading: Sparking <br class="brmob">a privacy revolution

```

Comment:
This should be translated too. ignore `<br class="brmob">`

Reply:
done in last commit